packages/fxa-auth-server/scripts/convert-customers-to-stripe-automatic-tax/convert-customers-to-stripe-automatic-tax.ts

```diff
@@ -265,4 +312,17 @@ export class StripeAutomaticTaxConverter {
       report.nextInvoice,
     ];
   }
+
+  /**
+   * Appends the report to the output file
+   * @param report an array representing the report CSV
+   */
+  async writeReport(report: (string | number | null)[]) {
+    const reportCSV = report.join(',') + '\n';
+
+    await fs.promises.writeFile(this.outputFile, reportCSV, {
+      flag: 'a+',
+      encoding: 'utf-8',
+    });
```

Comment:
Neat! Does this clear the `this.outputFile` between runs? Or if I run the script 5&times; will it just keep appending?

Reply:
It will continue appending